application/views/questionAdministration/partial/groupView.php

```diff
@@ -0,0 +1,197 @@
+                'summaryText'     => gT('Displaying {start}-{end} of {count} result(s).') . ' ' . sprintf(
+                    gT('%s rows per page'),
+                    CHtml::dropDownList(
+                        'pageSize',
+                        $pageSize,
+                        Yii::app()->params['pageSizeOptions'],
+                        [
+                            'class' => 'changePageSize form-select',
+                            'style' => 'display: inline; width: auto'
+                        ]
+                    )
+                ),
+
+                // Columns to dispplay
+                'columns'         => [
+                    // Group Id
+                    [
+                        'header' => gT('Group ID'),
+                        'name'   => 'group_id',
+                        'value'  => '$data->gid',
+                        'filter'            => false,
+
+                    ],
+
+                    // Group Order
+                    [
+                        'header' => gT('Group order'),
+                        'name'   => 'group_order',
+                        'value'  => '$data->group_order',
+                        'filter'            => false,
+                    ],
+
+                    // Group Name
+                    [
+                        'header'      => gT('Group name'),
+                        'name'        => 'group_name',
+                        'value'       => '$data->primaryTitle',
+                        'htmlOptions' => ['class' => ''],
+                    ],
+
+                    // Description
+                    [
+                        'header'      => gT('Description'),
+                        'name'        => 'description',
+                        'type'        => 'raw',
+                        'value'       => 'viewHelper::flatEllipsizeText($data->primaryDescription, true, 0)',
+                        'htmlOptions' => ['class' => ''],
+                        'filter'            => false,
+                    ],
+                    // Action buttons (defined in model)
+                    [
+                        'header'      => gT('Action'),
+                        'name'        => 'actions',
+                        'type'        => 'raw',
+                        'filter'            => false,
+                        'value'       => '$data->buttons',
+                        'headerHtmlOptions' => ['class' => 'ls-sticky-column'],
+                        'htmlOptions'       => ['class' => 'text-center button-column ls-sticky-column'],
+                    ],
+
+                ],
+                'ajaxUpdate'      => 'question-group-grid',
+                'afterAjaxUpdate' => 'bindPageSizeChange'
+            ]
+        );
+        ?>
+    </div>
+</div>
+
+<!-- To update rows per page via ajax -->
+<?php App()->getClientScript()->registerScript("ListQuestionGroups-pagination", "
+        var bindPageSizeChange = function(){
```

Comment:
If you use PHP heredoc notation instead of string, you'll get nice syntax highlight. :)

https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc

```php
$string = <<<JAVASCRIPT
let a = 10;
JAVASCRIPT;

Reply:
Thanks. This is really cool !